services/intersection-api/api/src/main/java/us/dot/its/jpo/ode/api/services/KafkaProducerService.java

```diff
@@ -0,0 +1,41 @@
+package us.dot.its.jpo.ode.api.services;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.kafka.core.KafkaTemplate;
+import org.springframework.stereotype.Service;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import us.dot.its.jpo.geojsonconverter.partitioner.RsuIntersectionKey;
+import us.dot.its.jpo.ode.api.models.snmp.RsuState;
+
+@Service
+public class KafkaProducerService {
+
+    private final KafkaTemplate<String, String> kafkaTemplate;
+
+    private ObjectMapper objectMapper = new ObjectMapper();
+
+    private static final Logger logger = LoggerFactory.getLogger(KafkaProducerService.class);
+
+    public KafkaProducerService(KafkaTemplate<String, String> kafkaTemplate) {
```

Comment:
(suggestion) use a spring injected object mapper rather than creating a new one.

```suggestion
    public KafkaProducerService(KafkaTemplate<String, String> kafkaTemplate, ObjectMapper objectMapper) {
        this.objectMapper = objectMapper;
```